src/utils/formik-helpers.ts

```diff
@@ -112,26 +113,65 @@ export const touchedDescriptionTabFields: FormikTouched<Publication> = {
   },
 };
 
-export const touchedReferenceTabFields: FormikTouched<Publication> = {
-  entityDescription: {
-    reference: {
-      publicationContext: {
-        type: true,
-        title: true,
-      },
-      publicationInstance: {
-        type: true,
-        articleNumber: true,
-        issue: true,
-        pages: {
-          begin: true,
-          end: true,
+export const touchedReferenceTabFields = (publicationType: PublicationType | ''): FormikTouched<any> => {
```

Comment:
enten så må denne returtypen være `FormikTouched<any>` eller så må vi opprette Publication-typer for alle de forskjellige typene. Tror det er lurt å definere alle de forskjellige publication-typene etterhvert, men lurer på om vi skal vente litt med det? Eller skal jeg bare starte nå? hva tror du, @elkebab?

Reply:
hm... den bør i hvert fall bli typet så vi vet at feltene vi bruker her ikke er helt på jordet. 
`PublicationInstance` bør kanskje bli `Journal | Book | Repost | xxx` den..? frykter det vil gi mer kluss andre steder igjen da, så kan godt ta det i en egen PR i hvert fall

Reply:
frykter det bare vil bli vondere jo lenger vi venter også 😬

Reply:
Ja, både `PublicationContext` og `PublicationInstance` har forskjellig innhold basert på type. Kanskje jeg bare skal ta den jobben nå da? Men at jeg tar det etter denne PRen, men før jeg begynner på `Report`-typen?

Reply:
kan sikkert være like greit det ja